spyder_kernels/comms/frontendcomm.py

```diff
@@ -124,6 +124,35 @@ def cache_message(self, comm_id, msg):
         self._cached_messages[comm_id].append(msg)
 
     # --- Private --------
+    def _check_comm_reply(self):
+        """
+        Send comm message to frontend to check if the iopub channel is ready
+        """
+        if len(self._pending_comms) == 0:
+            return
+        for comm in self._pending_comms.values():
+            self._notify_comm_ready(comm)
+        self.kernel.io_loop.call_later(1, self._check_comm_reply)
+
+    def _notify_comm_ready(self, comm):
+        """Send messages about comm readiness to frontend."""
+        self.remote_call(
+            comm_id=comm.comm_id,
+            callback=self._comm_ready_callback
+        )._comm_ready()
+
+    def _comm_ready_callback(self, ret):
+        """A comm has replied"""
```

Comment:
```suggestion
        """A comm has replied, so process all cached messages related to it."""
```

I think that's what this is doing, right?